exonum/src/runtime/dispatcher/error.rs

```diff
@@ -51,6 +51,8 @@ pub enum Error {
     UnauthorizedCaller = 11,
     /// Malformed arguments for calling a service interface method.
     MalformedArguments = 12,
+    /// The service instance is already pending.
```

Comment:
Nit: IMO, something like "service instance is already transitioning to a new status" would be more understandable.